### PR TITLE
add BoxAnatomy Component

### PR DESCRIPTION
**Pull request by javier-eche**

### CODE REVIEW

File: src/components/BoxHeight.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    return (
+        <>
+            <div class="panel is-primary">
+                <p class="panel-heading">{props.theHeight}</p>
+                <a class="panel-block is-active" href="#">
+                    <span class="panel-icon">
+                        <i class="fas fa-book" aria-hidden="true"></i>
+                    </span> {props.theHeight}
+                </a>
+            </div>
```
Comment (AlexisEche) [inline src/components/BoxHeight.js]: Use className instead class

File: src/components/BoxHeight.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const BoxHeight = (props) => {
+
+    return (
+        <>
```
Comment (AlexisEche) [inline src/components/BoxHeight.js]: Fragment is not necesary , you have  only one child.

File: src/PokedexApp.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import './Pokedex.sass'
+import BoxAnatomy from './components/BoxAnatomy';
 
 const PokedexApp = () => {
   return (
-    <h1>Poke</h1>
+    <>
```
Comment (AlexisEche) [inline src/PokedexApp.js]: Fragment is not necesary , you have only one child.

**Review by AlexisEche — CHANGES_REQUESTED** [review]
Good job , but there are think could improve :D

File: src/PokedexApp.js [hunk 4]
```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import './Pokedex.sass'
+import BoxAnatomy from './components/BoxAnatomy';
 
 const PokedexApp = () => {
   return (
-    <h1>Poke</h1>
+
```
Comment (joel-eche) [inline src/PokedexApp.js]: You could erase this empty space.

File: src/PokedexApp.js [hunk 5]
```diff
@@ -1,9 +1,12 @@
...
+      <BoxAnatomy />
+
```
Comment (joel-eche) [inline src/PokedexApp.js]: You could erase this empty space.

File: src/components/BoxHeight.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+            <div className="panel is-primary">
+                <p className="panel-heading">{props.theHeight}</p>
+                <a className="panel-block is-active" href="#">
+                    <span className="panel-icon">
+                        <i className="fas fa-book" aria-hidden="true"></i>
+                    </span> {props.theHeight}
+                </a>
+            </div>
+    );
+}
```
Comment (joel-eche) [inline src/components/BoxHeight.js]: Please, use indentation with 2 spaces.

**Review by joel-eche — CHANGES_REQUESTED** [review]
Try to follow good practices.

File: src/components/BoxHeight.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+const BoxHeight = (props) => {
+
+    return (
+      <div className="panel is-primary">
+        <p className="panel-heading">{props.theHeight}</p>
+          <a className="panel-block is-active" href="#">
+            <span className="panel-icon">
+              <i className="fas fa-book" aria-hidden="true"></i>
+                </span> {props.theHeight}
```
Comment (joel-eche) [inline src/components/BoxHeight.js]: Please flollow indentation.

**Review by joel-eche — CHANGES_REQUESTED** [review]
Take care of indentation :cat:

**Pull request merged**